<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="ca_le">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="CaBtn" name="ac_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>lightgray</string>
     </property>
     <property name="role" stdset="0">
      <string>ac</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CaBtn" name="zf_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>lightgray</string>
     </property>
     <property name="role" stdset="0">
      <string>zf</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CaBtn" name="bf_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>lightgray</string>
     </property>
     <property name="role" stdset="0">
      <string>bf</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="CaBtn" name="o4_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>orange</string>
     </property>
     <property name="role" stdset="0">
      <string>op</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="CaBtn" name="n7_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CaBtn" name="n8_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="CaBtn" name="n9_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="CaBtn" name="o3_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>orange</string>
     </property>
     <property name="role" stdset="0">
      <string>op</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="CaBtn" name="n4_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="CaBtn" name="n5_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="CaBtn" name="n6_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="CaBtn" name="o1_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>orange</string>
     </property>
     <property name="role" stdset="0">
      <string>op</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="CaBtn" name="n1_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="CaBtn" name="n2_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="CaBtn" name="n3_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="CaBtn" name="o2_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>orange</string>
     </property>
     <property name="role" stdset="0">
      <string>op</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="CaBtn" name="nd_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="CaBtn" name="ca_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>orange</string>
     </property>
     <property name="role" stdset="0">
      <string>ca</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="CaBtn" name="n0_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="bg" stdset="0">
      <string>gray</string>
     </property>
     <property name="role" stdset="0">
      <string>num</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CaBtn</class>
   <extends>QPushButton</extends>
   <header location="global">ca_btn</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
